contrib/for-tests/network-tester/slow-pod.json

```diff
@@ -0,0 +1,28 @@
+{
+  "kind": "Pod",
+  "apiVersion": "v1",
```

Comment:
This should now go into:
https://github.com/kubernetes/kubernetes/tree/master/test/images/network-tester